lib/plugins/create/templates/README.md

```diff
@@ -49,3 +47,15 @@ afterwards.
 5. `serverless deploy`
 6. `serverless invoke --function hello --path event.json`
 7. `serverless remove`
+
+## Golang
+
+Follow these simple steps to create and deploy your golang service, run your function and remove the service afterwards.
+
+1. install latest version of `serverless` and `go`
+2. `mkdir my-first-service && cd my-first-service`
+3. `serverless create --template aws-golang`
+4. `GOOS=linux GOARCH=amd64 go build -o hello hello.go`
```

Comment:
Thats just standard Go Build stuff. If you have a build tool defined for example it would take care of this. But for a Go developer this is not surprising and totally standard.